include/highfive/H5PropertyList.hpp

```diff
@@ -45,10 +45,8 @@ class PropertyList {
   public:
     ~PropertyList();
 
-#ifdef H5_USE_CXX11
```

Comment:
It was buggy, `H5_USE_CXX11` is always **defined** to `0` or to `1`, but **defined**.